### ISSUE TITLE
More accurate detection of active mobile users

### ISSUE THREAD

**Issue by rfk** [mozilla/fxa-activity-metrics#8]
Wasn't sure where to file this, here seems as good a place as any to start...

Our definition of an "active" user is a user who generates an `account.signed` event on any given day.  On Android, the sync process runs under the control of the operating system rather than directly from Firefox.  What this means is that, if you've ever set up Sync on Firefox for Android, then it will continue syncing _until you explicitly disconnect it_, even if you long since stopped actually using Firefox on that device.

I've no idea how much this might skew our activity, retention, or multi-device-usage numbers.

I'm also not sure how to proceed.  We may need to find a better event to indicate sync activity on mobile, e.g. trying to detect "actually uploaded some changed data to sync" as a better proxy for "was actually using Firefox on this device".  @davismtl thoughts?

**Comment by davismtl** [comment]
@rfk this definition sounds pretty accurate to me. As soon as they visit any website, their history will have content that will be synced. 

The only question I wonder is if that is accurate enough. I'm not familiar with the frequency of Sync on Android. If we are off by a few hours, no biggie. If we are off by a few days, then the data might be too inaccurate.

**Comment by rfk** [comment]
> their history will have content that will be synced. 

Doesn't help us if they're not syncing history of course, but it's a start :-)

**Comment by davismtl** [comment]
Good point. I think it's the one that gets disabled the most. :-/

Do we know how often new data gets synced (DAU/MAU)? Can we compare that to the DAU/MAU ratio of Android? If the sync numbers look smaller, it might mean it's a bad data point.

**Comment by shane-tomlinson** [comment]
@rfk - can you triage this?

**Comment by rfk** [comment]
I don't think we can do much about this in the short term, so I'm going to remove it from the funnel-metrics milestone and put it in the backlog.

**Comment by vladikoff** [comment]
Removing 2018 label
